docs/cpu-specific-build-macros.md

```diff
@@ -60,6 +60,12 @@ For Cortex-A53, following errata build flags are defined :
      CPU. This needs to be enabled only for revision <= r0p3 of the CPU. From
      r0p4 and onwards, this errata is enabled by default.
 
+*   `ERRATA_A53_855873`: This applies errata 855873 workaround to Cortex-A53
+     CPUs. Though the erratum is present in every revision of the CPU,
+     this workaround is only applied to CPUs from r0p3 onwards, which feature
+     a chicken bit in CPUACTLR_EL1 to enable a hardware workaround.
+     Earlier revisions of the CPU are meant to be patched by the rich OS.
```

Comment:
Noob here: what is rich OS? only other usage seems to be in plat/xilinx/zynqmp/pm_service/pm_svc_main.c -> did we mean Secure OS like OPTEE?